oauth2client/_helpers.py

```diff
@@ -11,12 +11,216 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Helper functions for commonly used utilities."""
 
 import base64
+import functools
+import inspect
 import json
+import logging
+import os
+import warnings
 
 import six
+from six.moves import urllib
+
+
+__author__ = [
```

Comment:
One of the many reasons I despise author attribution in code is that there's never a good answer for what to do when a file is modified like we're modifying this file.

I'm not sure of the implications of removing it, so I can't tell you to remove it, so... either keep it as-is or add yourself and the authors of the preëxisting content in `_helpers.py` to it?